docs/_newsfragments/1583.breakingchange.rst

```diff
@@ -0,0 +1,2 @@
+The class :class:`~.falcon.request_helpers.BoundedStream` was deprecated.
```

Comment:
Rather than saying the class itself is deprecated, it may be clearer to reword this note to say that *referencing* the class through the old module is deprecated.

Reply:
Okay. I will do this.